### PR TITLE
Only show global most popular container within the info section

### PR DESCRIPTION
**Pull request by phamann**
As requested form editorial and central production: they don't want the section specific content to appear on any content within the info section.

/cc @gklopper

### CODE REVIEW

File: dev-build/conf/routes [hunk 1]
```diff
@@ -203,7 +203,6 @@ GET         /metrics/afg
 GET         /radiator                                                                                                controllers.admin.RadiatorController.renderRadiator()
 GET         /radiator/pingdom                                                                                        controllers.admin.RadiatorController.pingdom()
 GET         /radiator/commit/*hash                                                                                   controllers.admin.RadiatorController.commitDetail(hash)
-GET         /troubleshoot/football                                                                                   controllers.admin.FootballTroubleshooterController.renderFootballTroubleshooter()
```
Comment (phamann) [inline dev-build/conf/routes]: This broke dev-build since @rich-nguyen's changes in #8848

**Comment by gklopper** [comment]
arghhhh - you know I'm not going to like this one :frowning:  `if (section == "...") {...}` is a slippery slope.

Can it hold for today and we chat about it tomorrow?

**Comment by gklopper** [comment]
:+1: - fixed my worries with a vaguely threatening comment (just let it run tests).

**Comment by phamann** [comment]
:laughing:

File: dev-build/conf/routes (hunk 1 above)
Comment (rich-nguyen) [inline dev-build/conf/routes]: Why remove football?

Comment (rich-nguyen) [inline dev-build/conf/routes]: Oh i see. will fix

**Pull request merged**